generated_validate_yamls/conda.python.generated.yaml

```diff
@@ -11,8 +11,6 @@ files:
 - Lib/site-packages/conda_env-*.egg-info
 - Lib/site-packages/conda_env-*.egg-info/**/*
 - Lib/site-packages/conda_env/**/*
-- Lib/site-packages/test_data/__init__.py
-- Lib/site-packages/test_data/env_metadata/__init__.py
```

Comment:
@CJ-Wright I had to code around conda's shipping of test data here too. 🤦

Reply:
yep :/